web/templates/input_pill.hbs

```diff
@@ -1,4 +1,8 @@
-<div class='pill {{#if deactivated}} deactivated-pill {{/if}}'{{#if user_id}}data-user-id="{{user_id}}"{{/if}}{{#if group_id}}data-user-group-id="{{group_id}}"{{/if}}{{#if stream_id}}data-stream-id="{{stream_id}}"{{/if}} tabindex=0>
+<div class='pill {{#if deactivated}} deactivated-pill {{/if}}'
+  {{#if user_id}}data-user-id="{{user_id}}"{{/if}}
+  {{#if group_id}}data-user-group-id="{{group_id}}"{{/if}}
+  {{#if stream_id}}data-stream-id="{{stream_id}}"{{/if}}
+  {{#if data_syntax}}data-syntax="{{data_syntax}}"{{/if}} tabindex=0>
```

Comment:
Let's use [Handlebars whitespace control](https://handlebarsjs.com/guide/expressions.html#whitespace-control), to remove the need to modify the tests ahead in the commit to account for these newline characters.